taxonomies/categories.txt

```diff
@@ -44363,8 +44293,8 @@ protected_name_type:en: pdo
 origins:en: en:Spain
 
 <en:Berries
-en:Blackcurrants, Raw black currant
-fr:Cassis, Cassis cru
+en:Blackcurrants, black currant
+fr:Cassis, Cassis
```

Comment:
There are many entries like that with the "raw" synonym now identical to the main entry in English and/or French